autoload/yankitute.vim

```diff
@@ -75,3 +77,11 @@ function! s:eval(results, replace) abort
     call add(a:results, eval(a:replace[2:]))
     return submatch(0)
 endfunction
+
+function! s:expand_returns(lst) abort
+  let arr = []
```

Comment:
Nitpick, but for consistent style please use the `l:` prefix for local variables.